db/demo-site-data/users.yml

```diff
@@ -0,0 +1,26 @@
+---
+User_1:
+  id: 1
+  username: admin
+  email: admin@example.com
+  encrypted_password: "$2a$11$BuIQa84a0DBc9vAuJ.PubOb3K7L8.UlMcOPltxRGU9j2FfIQkNolC"
```

Comment:
This is public. I wanted to highlight in case this hash protects a sensitive value that should not be exposed, even hashed.

Reply:
Although this is for a demo site, consider generating unique random values for sensitive data (password, confirmation_token + ?) to mitigate risk of demo sites being compromised by a reused password/token. 

Perhaps there's a possibility the demo admin user is not deleted/reset when a demo site is converted to a working site?

Reply:
At some point I want to add a curl-able install.sh script, so I'll have that prompt people to choose their own username and password for the admin account.